### PR TITLE
Progress till 05-03-22

### PR DESCRIPTION
**Pull request by sadab-halim**

### CODE REVIEW

File: Sadab-Halim/25-02-22/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+### Dockerfile
+- Dockerfile is a text file written in a specific file that docker can understand. 
+- It consist of two things
+  1. Instructions
+  2. Arguments
+
+- Everything written in the UPPERCASE on the left most side is an instruction.
+- Docker builds the image in layered architecture. 
+- If we wish to see the layer then you can `run docker history <image-name>`
```
Comment (kranurag7) [inline Sadab-Halim/25-02-22/README.md:44]:
```suggestion
- If we wish to see the layer then you can `docker history <image-name>`
```

File: Sadab-Halim/03-03-22/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Docker from [Docker for Beginners: Full Course](https://www.youtube.com/watch?v=zJ6WbK9zFpI)
+
+Learnt about Orechstration <br/>
```
Comment (kranurag7) [inline Sadab-Halim/03-03-22/README.md:3]:
```suggestion
Learnt about Orchestration <br/>
```

**Review by kranurag7 — CHANGES_REQUESTED** [review]

**Review by sadab-halim — COMMENTED** [review]
Thanks for the correction

**Pull request merged**